OpenDreamRuntime/Procs/DMProc.cs

```diff
@@ -827,9 +827,9 @@ public DreamValue GetReferenceValue(DreamReference reference, bool peek = false)
                 case DMReference.Type.ListIndex: {
                     GetIndexReferenceValues(reference, out var index, out var indexing, peek);
 
-                    ProcStatus subState = GetIndex(indexing, index, this, out Result);
+                    ProcStatus subState = GetIndex(indexing, index, this, out var result);
```

Comment:
maybe best to change the var name entirely? `indexResult` or something?

also whoops :trollface:

Reply:
Would be a good change